examples/machine-set.yaml

```diff
@@ -5,22 +5,22 @@ metadata:
   name: aws-actuator-testing-machine
   namespace: test
   labels:
-    sigs.k8s.io/cluster-api-cluster: tb-asg-35
-    sigs.k8s.io/cluster-api-machine-role: infra
-    sigs.k8s.io/cluster-api-machine-type: master
+    machine.openshift.io/cluster-api-cluster: tb-asg-35
+    machine.openshift.io/cluster-api-machine-role: infra
+    machine.openshift.io/cluster-api-machine-type: master
 spec:
   replicas: 2
   selector:
     matchLabels:
-      sigs.k8s.io/cluster-api-machineset: test-master
-      sigs.k8s.io/cluster-api-cluster: tb-asg-35
+      machine.openshift.io/cluster-api-machineset: test-master
```

Comment:
for consistency, should this be `machine-api-machineset` and same for all cluster-api references?

Reply:
Don't have any opinion on that, maybe some others can chime in here.

Reply:
![itdoesnotmatter](http://www.quickmeme.com/img/46/46427613a99fa4706d302369a421f30c70f40f1f7235016ce95552c95d9883af.jpg)

because it's only for convenience. Though, if it does, go with `machine.openshift.io/machineset-id`

Reply:
I agree with @ingvagabund I guess. It doesn't really matter. There's an argument to be made for keeping it exactly like upstream except for the domain prefix though. If I were picking anything I wanted, I'd call it `machine.openshift.io/machineset` I think. I don't know... maybe this isn't helpful at all.